sklearn/decomposition/pca.py

```diff
@@ -138,7 +142,40 @@ class PCA(BaseEstimator, TransformerMixin):
         Whitening will remove some information from the transformed signal
         (the relative variance scales of the components) but can sometime
         improve the predictive accuracy of the downstream estimators by
-        making there data respect some hard-wired assumptions.
+        making their data respect some hard-wired assumptions.
+
+    svd_solver : string {'auto', 'full', 'arpack', 'randomized'}
+        auto :
+            the solver is selected by a default policy based on X.shape and
```

Comment:
It might be useful to have a one-liner about what this policy is. (For people like me who are curious and are lazy to look at the code :-) )